Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -138,15 +138,10 @@ void GameCubePane::CreateWidgets()
   }
 
   // Add SP1 devices
-  for (const auto device : {
-           EXIDeviceType::None,
-           EXIDeviceType::Dummy,
-           EXIDeviceType::Ethernet,
-           EXIDeviceType::EthernetXLink,
-           EXIDeviceType::EthernetTapServer,
-           EXIDeviceType::EthernetBuiltIn,
-           EXIDeviceType::ModemTapServer,
-       })
+  for (const auto device :
+       {EXIDeviceType::None, EXIDeviceType::Dummy, EXIDeviceType::Ethernet,
+        EXIDeviceType::EthernetXLink, EXIDeviceType::EthernetTapServer,
+        EXIDeviceType::EthernetBuiltIn, EXIDeviceType::ModemTapServer, EXIDeviceType::Baseboard})
```

Comment:
```suggestion
        EXIDeviceType::EthernetBuiltIn, EXIDeviceType::ModemTapServer, EXIDeviceType::Baseboard,})
```
A terminating comma will let clang-format maintain the original formatting.